test/java/src/com/ibm/streamsx/topology/test/api/LowLatencyTest.java

```diff
@@ -90,7 +90,7 @@ public void threadedPortTest() throws Exception{
                 lowLatencyTag = (String) placement.get(JOperator.PLACEMENT_LOW_LATENCY_REGION_ID);
             String kind = (String)op.get("kind");
             JSONObject queue = (JSONObject) op.get("queue");
-            if(queue != null && (lowLatencyTag!=null || lowLatencyTag.equals(""))){
+            if(queue != null && (lowLatencyTag == null || lowLatencyTag.equals(""))){
```

Comment:
Replacing `equals("")` with i`sEmpty()`:

1. `if (queue != null && (lowLatencyTag == null || lowLatencyTag.isEmpty()))`
2. `if (queue != null && (lowLatencyTag != null && !lowLatencyTag.isEmpty()))`

I'm not familiar with the graph model and cannot decide: which alternative makes more sense, (1) or (2)?